tests/acceptance-tests/test/specs/smoke-tests/simple-login.app-test.js

```diff
@@ -21,9 +23,9 @@ describe("simple login test", function () {
     const usrData = yaml.safeLoad(fs.readFileSync("./tests/acceptance-tests/config/user-data.yml", "utf8"));
 
     // default to process env if we've got that
-    const adminEmail = process.env.REACTION_EMAIL || usrData.admin_email;
```

Comment:
Please check this line @Dannytebj , You removed the ` || usrData.admin_pw` which some of us are using from the userData.yml file. It will be good if you add this before merging